source/documentation/using_the_api/api_reference.md

```diff
@@ -225,19 +241,26 @@ Example response:
    },
 ```
 
-### `GET /entries`
+### <a name="get-entries">Get all entries from a register</a>
 
-Get all entries from the register. For example, all updates there have ever been to the `local-authority-eng` register.
+Endpoint: `GET /entries/`
 
-> Results from this API call are paginated. This call will return the first 100 entries from the first page of the register. Use `limit` to define the maximum number of entries you want and `start` to define the entry number you want to start from (in ascending order).
+Parameters: 
+* start (Optional): Collection page number. Defaults to ?.
+* limit (Optional): Collection page size. Defaults to ?. Maximum is ?. 
```

Comment:
Based on the above, this defaults to 100. @michaelabenyohai, @gidsg what's the maximum for entries?